app/controllers/concerns/verify_setup_completed.rb

```diff
@@ -0,0 +1,36 @@
+module VerifySetupCompleted
+  extend ActiveSupport::Concern
+
+  MANDATORY_CONFIGS = %i[
+    community_description
+    community_member_description
+    community_action
+    tag_url
```

Comment:
@citizen428 I'm getting an error on tag_url, was this meant to be tagline. I don't see a site config variable called tag_url?

Reply:
🤦 Yes, that was most certainly mean to be `tagline`, sorry for that.